orb/commands/@drupal.yml

```diff
@@ -111,14 +112,17 @@ drupal-helm-deploy:
             extra_noauthips="--set nginx.noauthips.vpn=${VPN_IP}/32"
           fi
 
-          if [[ ! -z "<<parameters.chart_version>>" ]] ; then
+          if [[ "<<parameters.chart_version>>" == "latest" ]] ; then
+            version="--version 0.3.7"
```

Comment:
I like having a string with a specific meaning as the default rather than an empty value, but if the version is "latest", shouldn't we skip the tag altogether? There are advantages to having the orb and the chart in sync by default, but then "latest" is not a good description of the default value.

Reply:
i'm trying to offload chart version fixating from certain in favor of this. I should probably ask if they need it altogether.

Reply:
alright, they don't need it
> It was only needed there because there wasn't default value so deployment failed. Great if it can be removed from there 
setting to ""